### PR TITLE
Fix HttpError (415) on login and adjust .gitignore rule for creds.json

### PR DESCRIPTION
**Pull request by generic-user1**
###  TL; DR

1. Fixed `HttpError` when logging into Jellyfin 10.7
2. Updated .gitignore rule for creds.json to apply globally in the repository

### Login Error

When attempting to log into Jellyfin Server 10.7.7 (and possibly other versions), Jellyfin-CLI would crash with an unhandled `HttpError`; specifically `415 Unsupported Media Type`.

More precisely, `HttpClient.login(username, password)` would encounter this error because when forming the POST to log in, credentials were passed using the generic `data` parameter of `aiohttp.ClientSession.post()`. Using the `json` parameter instead fixes the issue with very little effort.

### .gitignore changes

The .gitignore file for this repository has always included an entry for creds.json, but that entry only applied within a certain sub-directory of the project. However, creds.json appears to be created in the current working directory, wherever that may be. This could result in creds.json being accidentally included in the public repository. Replacing the old rule with rules that apply globally eliminates this issue.

### CODE REVIEW

**Comment by generic-user1** [comment]
Sample output demonstrating login issue before this change:
[jellyfin-cli.sample_login_415_error_output.txt](https://github.com/marios8543/Jellyfin-CLI/files/7819429/jellyfin-cli.sample_login_415_error_output.txt)

**Comment by marios8543** [comment]
well i hope it works lmao
